spec/fixtures/files/profile_fields.csv

```diff
@@ -1,4 +1,4 @@
+Full test,text_field,Test,Test field,Basic,header,true
```

Comment:
I update the test CSV data to remove color fields and also because I restructured the tests a little bit.